diff -purN Python-2.7.2.orig/Lib/cgi.py Python-2.7.2/Lib/cgi.py
--- Python-2.7.2.orig/Lib/cgi.py	2011-11-02 18:01:39.000000000 +0100
+++ Python-2.7.2/Lib/cgi.py	2011-11-02 18:01:59.000000000 +0100
@@ -1,4 +1,4 @@
-#! /usr/local/bin/python
+#! /usr/bin/python
 
 # NOTE: the above "/usr/local/bin/python" is NOT a mistake.  It is
 # intentionally NOT "/usr/bin/env python".  On many systems
diff -purN Python-2.7.2.orig/Modules/Setup.dist Python-2.7.2/Modules/Setup.dist
--- Python-2.7.2.orig/Modules/Setup.dist	2011-11-02 18:01:42.000000000 +0100
+++ Python-2.7.2/Modules/Setup.dist	2011-11-02 18:04:51.000000000 +0100
@@ -34,7 +34,8 @@
 #
 # which defines a Make variable definition inserted into Makefile.in
 #
-# Finally, if a line contains just the word "*shared*" (without the
+# Finally, if a line contains just the word "*
+# shared*" (without the
 # quotes but with the stars), then the following modules will not be
 # built statically.  The build process works like this:
 #
@@ -153,7 +154,7 @@ GLHACK=-Dclear=__GLclear
 # modules are to be built as shared libraries (see above for more
 # detail; also note that *static* reverses this effect):
 
-#*shared*
+*shared*
 
 # GNU readline.  Unlike previous Python incarnations, GNU readline is
 # now incorporated in an optional module, configured in the Setup file
@@ -390,7 +391,7 @@ GLHACK=-Dclear=__GLclear
 #
 # First, look at Setup.config; configure may have set this for you.
 
-#dbm dbmmodule.c 	# dbm(3) may require -lndbm or similar
+dbm dbmmodule.c -DHAVE_NDBM_H -lgdbm_compat -lgdbm # dbm(3) may require -lndbm or similar
 
 # Anthony Baxter's gdbm module.  GNU dbm(3) will require -lgdbm:
 #
