--- mysql-5.6.23/sql-common/client_plugin.c.orig	2015-04-02 13:08:28.255818357 +0200
+++ mysql-5.6.23/sql-common/client_plugin.c	2015-04-02 13:11:57.325709613 +0200
@@ -259,6 +259,7 @@
 {
   MYSQL mysql;
   struct st_mysql_client_plugin **builtin;
+  va_list dummy;
 
   if (initialized)
     return 0;
@@ -275,7 +276,7 @@
   mysql_mutex_lock(&LOCK_load_client_plugin);
 
   for (builtin= mysql_client_builtins; *builtin; builtin++)
-    add_plugin_noargs(&mysql, *builtin, 0, 0);
+    add_plugin_noargs(&mysql, *builtin, 0, 0, dummy);
 
   mysql_mutex_unlock(&LOCK_load_client_plugin);
 
@@ -319,6 +320,7 @@
 mysql_client_register_plugin(MYSQL *mysql,
                              struct st_mysql_client_plugin *plugin)
 {
+  va_list dummy;
   if (is_not_initialized(mysql, plugin->name))
     return NULL;
 
@@ -333,7 +335,7 @@
     plugin= NULL;
   }
   else
-    plugin= add_plugin_noargs(mysql, plugin, 0, 0);
+    plugin= add_plugin_noargs(mysql, plugin, 0, 0, dummy);
 
   mysql_mutex_unlock(&LOCK_load_client_plugin);
   return plugin;
