--- chromium-30.0.1599.114/third_party/icu/icu.gyp.orig	2013-10-22 02:11:03.000000000 +0200
+++ chromium-30.0.1599.114/third_party/icu/icu.gyp	2013-10-22 02:14:00.000000000 +0200
@@ -27,7 +27,7 @@
       }],
       ['(OS=="linux" or OS=="freebsd" or OS=="openbsd" or OS=="solaris" \
          or OS=="netbsd" or OS=="mac" or OS=="android") and \
-        (target_arch=="arm" or target_arch=="ia32" or \
+        (target_arch=="ia32" or \
          target_arch=="mipsel")', {
         'target_conditions': [
           ['_toolset=="host"', {
