--- chromium-30.0.1599.114/third_party/webrtc/common_audio/common_audio.gyp.orig	2013-11-07 09:49:02.000000000 +0100
+++ chromium-30.0.1599.114/third_party/webrtc/common_audio/common_audio.gyp	2013-11-07 09:50:26.000000000 +0100
@@ -103,7 +103,7 @@
             'signal_processing/spl_sqrt_floor.c',
           ],
           'conditions': [
-            ['armv7==1', {
+            ['armv7==1 and arm_neon==1', {
               'dependencies': ['common_audio_neon',],
               'sources': [
                 'signal_processing/filter_ar_fast_q12_armv7.S',
@@ -150,7 +150,7 @@
         },
       ],  # targets
     }],
-    ['target_arch=="arm" and armv7==1', {
+    ['target_arch=="arm" and armv7==1 and arm_neon==1', {
       'targets': [
         {
           'target_name': 'common_audio_neon',
--- chromium-30.0.1599.114/third_party/webrtc/modules/audio_processing/audio_processing.gypi.orig	2013-11-07 09:49:14.000000000 +0100
+++ chromium-30.0.1599.114/third_party/webrtc/modules/audio_processing/audio_processing.gypi	2013-11-07 09:59:02.000000000 +0100
@@ -119,7 +119,7 @@
         ['target_arch=="ia32" or target_arch=="x64"', {
           'dependencies': ['audio_processing_sse2',],
         }],
-        ['target_arch=="arm" and armv7==1', {
+        ['target_arch=="arm" and armv7==1 and arm_neon==1', {
           'dependencies': ['audio_processing_neon',],
         }],
       ],
@@ -161,7 +161,7 @@
         },
       ],
     }],
-    ['target_arch=="arm" and armv7==1', {
+    ['target_arch=="arm" and armv7==1 and arm_neon==1', {
       'targets': [{
         'target_name': 'audio_processing_neon',
         'type': 'static_library',
--- chromium-30.0.1599.114/third_party/webrtc/modules/audio_coding/codecs/isac/fix/source/isacfix.gypi.orig	2013-11-07 09:49:27.000000000 +0100
+++ chromium-30.0.1599.114/third_party/webrtc/modules/audio_coding/codecs/isac/fix/source/isacfix.gypi	2013-11-07 10:00:06.000000000 +0100
@@ -72,7 +72,7 @@
             'WEBRTC_LINUX',
           ],
         }],
-        ['target_arch=="arm" and armv7==1', {
+        ['target_arch=="arm" and armv7==1 and arm_neon==1', {
           'dependencies': [ 'isac_neon', ],
           'sources': [
             'lattice_armv7.S',
@@ -87,7 +87,7 @@
     },
   ],
   'conditions': [
-    ['target_arch=="arm" and armv7==1', {
+    ['target_arch=="arm" and armv7==1 and arm_neon==1', {
       'targets': [
         {
           'target_name': 'isac_neon',
